src/package.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
+        <members>Pan_Card__c</members>
+        <members>StudentSubjects__c</members>
+        <members>Subject__c</members> 
+        <members>Teacher__c</members> 
+        <members>Year_of_Class__c</members>
+        <members>College__c</members>
+        <members>Employee__c</members>
+        <name>CustomObject</name>
+    </types>
+    <types>
+        <members>Pan_Card__c</members>
+        <members>StudentSubjects__c</members>
+        <members>Subject__c</members> 
+        <members>Teacher__c</members> 
+        <members>Year_of_Class__c</members>
+        <members>College__c</members>
+        <members>Employee__c</members>
+        <name>CustomTab</name>
+    </types>
+    <types>
+        <members>Contact.Pan_Card__c</members>
```

Comment:
There is no Contact.object and Lead.object files in the Objects folder. please Add both files for custom fields.